src/coreclr/jit/emit.cpp

```diff
@@ -7794,25 +7799,32 @@ void emitter::emitOutputDataSec(dataSecDsc* sec, BYTE* dst)
 //
 // Arguments:
 //    section - the data section description
+//    dst     - address of the data section
 //
 // Notes:
 //    The output format attempts to mirror typical assembler syntax.
 //    Data section entries lack type information so float/double entries
 //    are displayed as if they are integers/longs.
 //
-void emitter::emitDispDataSec(dataSecDsc* section)
+void emitter::emitDispDataSec(dataSecDsc* section, BYTE* dst)
 {
     printf("\n");
 
     unsigned offset = 0;
 
     for (dataSection* data = section->dsdList; data != nullptr; data = data->dsNext)
     {
+        if (emitComp->opts.disAddr)
+        {
+            printf("; @" FMT_ADDR "\n", DBG_ADDR(dst));
+        }
+
```

Comment:
Is there an example of what this displayed previously vs what it displays now?

Reply:
+1

Reply:
If you set `COMPlus_JitDasmWithAddress=1` you'll see:

```
; @ 000002cc`0d8d4fc4 
RWD00  	dq	0020002000200020h, 0020002000200020h
; @ 000002cc`0d8d4fd4 
RWD16  	dq	006200720065006Bh, 0073006F00720065h
; @ 000002cc`0d8d4fe4 
RWD32  	dq	006F00670065006Eh, 0074006100690074h
; @ 000002cc`0d8d4ff4 
RWD48  	dq	0074006F00670065h, 0065007400610069h
```

where the `; @ ...` lines are new.

Reply:
I'd assume those be excluded in `DiffableDasm` (I've never used `JitDasmWithAddress`, didn't know it existed 👍)?

Also its worth noting that the address shown there is a multiple of `4`, but I would have expected it to be a multiple of `16` (should probably be `000002cc`0d8d4fd0`, not `000002cc`0d8d4fc4`). Was this done with `MIN_DATA_ALIGN` or is there a different bug here?